app/api/bookyt/api/bookings.rb

```diff
@@ -0,0 +1,31 @@
+module Bookyt
+  class API
+    class Bookings < Grape::API
+      resource :bookings do
+        desc 'Create a new booking'
+        params do
+          requires :title, type: String, desc: 'Title of the booking'
+          requires :amount, type: Float, desc: 'Amount to be booked'
+          requires :value_date, type: Date, desc: 'Date of the booking'
+          requires :credit_account_tag, type: String, desc: 'Tag of the credit account'
+          requires :debit_account_tag, type: String, desc: 'Tag of the debit account'
+          optional :comments, type: String, desc: 'Additional comments'
+        end
+        post do
+          credit_account = Account.tagged_with(params[:credit_account_tag]).first
```

Comment:
Simply taking the first seems a bit dangerous for an API. We should probably signal an error if we get more than one match.